cms/djangoapps/contentstore/views/item.py

```diff
@@ -805,8 +812,14 @@ def create_xblock_info(xblock, data=None, metadata=None, include_ancestor_info=F
         "format": xblock.format,
         "course_graders": json.dumps([grader.get('type') for grader in graders]),
         "has_changes": has_changes,
-        "override_type": override_type,
+        "actions": xblock_actions,
+        "explanatory_message": explanatory_message
     }
+
+    # Entrance exam subsection should be hidden.
+    if xblock.category == 'sequential' and getattr(parent_xblock, "is_entrance_exam", False):
+        xblock_info["is_visible"] = False
```

Comment:
Instead of "is_visible", @andy-armstrong and I prefer "is_header_visible". We both think that "is_visible" would refer to the children as well, and it is a general concept that we may wish to support in the future (thus we would like to reserve the property name "is_visible").